### DISCUSSION TITLE
Keeping one design variable changing and others constant.

### DISCUSSION THREAD

**Discussion by Atulsingh92** [anyoptimization/pymoo#249] [Q&A]
Hello, 

In the xl and xu declaration, is there a possiblity, to keep only one variable to update while keeping others as constant?
```
xl = anp.array([dv1_l, dv2_l,  dv3_l]),  #dvi_l  = design variable i _ lower
xu = anp.array([dv1_u, dv2_u, dv3_u]), # dvi_u = design variable i _ upper
```
The idea was to understand what happens to the objective, if dv1 is kept as a variable, while dv2 and dv3 are constant? Similarly for dv2 and dv3

Also, do correct me if I am wrong, if such a question is not suited for pymoo?

**Comment by blankjul** [comment]
That totally depends on your application what happens when you fix variables. For `pymoo` all variables are supposed to be modified. For (almost) fixing them do not set the lower equals to the upper bound, but leave a small epsilon (1e-16 or so) for variation.
Or do not even declare the value as a variable at all and define it as a constant in `_evaluate`.
